quickstarts/microsoft.network/azure-bastion/main.bicep

```diff
@@ -0,0 +1,86 @@
+@description('Name of new or existing vnet to which Azure Bastion should be deployed')
+param vnetName string = 'vnet01'
+
+@description('IP prefix for available addresses in vnet address space')
+param vnetIpPrefix string = '10.1.0.0/16'
+
+@allowed([
+  'new'
+  'existing'
+])
+@description('Specify whether to provision new vnet or deploy to existing vnet')
```

Comment:
I want to establish a pattern of putting the description decorator at the top so that the rest of them have some context when viewing the code...

Reply:
Looks like it is:

resource virtualNetwork 'Microsoft.Network/virtualNetworks@2020-05-01' **= if (vnetNewOrExisting == 'new'**) {

resource subnet 'Microsoft.Network/virtualNetworks/subnets@2020-05-01' = **if (vnetNewOrExisting == 'existing')** {

Is something missing or not right?